packages/components/select/src/select.vue

```diff
@@ -88,8 +88,7 @@
               ref="input"
               v-model="query"
               type="text"
-              class="el-select__input"
-              :class="[selectSize ? `is-${selectSize}` : '']"
+              :class="[ns.e('input'), ns.is(selectSize, !!selectSize)]"
```

Comment:
**Suggestion:** we can change `is` function to
```ts
const is = (name: string, state = true) =>
    name && state ? `${statePrefix}${name}` : ''
```

and
```suggestion
              :class="[ns.e('input'), ns.is(selectSize)]"
```

Reply:
You need change `is` function as well.

Reply:
fixed！